itsgracian/shoes
# update product  images and read products

## What does this Pull Request do?
Allow users to update the product images, read all product and single product

## Description of Task to be completed?
- I created a method which allows users to update the image
- I created another method which verifies if the product image field is not empty

### How should this be manually tested?
- open postman
- in URL put [http://localhost:5000/api/products/:productId/image/:/productImageId/update](http://localhost:5000/api/products/:productId/image/:/productImageId/update)
- opening a single product [http://localhost:5000/api/products/:slug](http://localhost:5000/api/products/:slug)
- the body should look like this
```
{
	"imageURL":"https://res.cloudinary.com/heza/image/hmdmukgw3ua5l0jffzlp.jpg",
	"color":"blue"
}

```
